make-dist

```diff
@@ -85,13 +85,33 @@ _python_autoselect() {
 build_dashboard_frontend() {
   CURR_DIR=`pwd`
   TEMP_DIR=`mktemp -d`
+
   $CURR_DIR/src/tools/setup-virtualenv.sh --python=$(_python_autoselect) $TEMP_DIR
   $TEMP_DIR/bin/pip install nodeenv
   $TEMP_DIR/bin/nodeenv -p --node=10.16.0
   cd src/pybind/mgr/dashboard/frontend
+
+  DEFAULT_LANG=`cat package.json | jq -r .config.locale`
+  if [ -z "$DASHBOARD_FRONTEND_LANGS" ]; then
+    BUILD_TARGET=":${DEFAULT_LANG}"
+  else
+    if [ "$DASHBOARD_FRONTEND_LANGS" == "ALL" ]; then
+      BUILD_TARGET=":*"
+    else
+      DASHBOARD_FRONTEND_LANGS=`echo "$DASHBOARD_FRONTEND_LANGS" | sed 's/ /,/g'`
```

Comment:
I'd use a new variable here. It's not a good practice to reuse/overwrite variables for holding data with different semantics (white-space separated vs. comma separated).